assets/client.go

```diff
@@ -184,6 +185,66 @@ func (c *TapdClient) GetAssetName(ctx context.Context,
 	return assetName, nil
 }
 
+// GetAssetPrice returns the price of an asset in satoshis. NOTE: this currently
+// uses the rfq process for the asset price. A future implementation should
+// use a price oracle to not spam a peer.
+func (c *TapdClient) GetAssetPrice(ctx context.Context, assetID string,
+	assetAmt, paymentMaxAmt uint64) (btcutil.Amount,
+	error) {
+
+	// We'll allow a short rfq expiry as we'll only use this rfq to
+	// gauge a price.
+	rfqExpiry := time.Now().Add(time.Minute).Unix()
+
+	// First we'll rfq a random peer for the asset.
```

Comment:
How does the random peer selection work? Do we have to specify a peer here? In regtest I see `LQDY: easy autoloop asset failed: id: 6edf1fbd38d326fa6272d56d82f3bfa043e2988fab78d82ddf2dbc64e3f7b72b, err: rpc error: code = Unknown desc = sell order peer must be specified`